client/src/infographic/bubble_definitions.js

```diff
@@ -28,34 +35,38 @@ const base_configs = [
     title: text_maker("financial_title"),
     description: text_maker("financial_desc"),
     enable_panel_filter: true,
+    svg_content: { svg: IconMoney },
   },
   {
     id: "people",
     title: text_maker("people_title"),
     description: text_maker("people_desc"),
     enable_panel_filter: true,
+    svg_content: { svg: IconEmployee },
   },
   {
     id: "results",
     title: text_maker("results_title"),
     description: text_maker("results_desc"),
+    svg_content: { svg: IconClipboard },
   },
   {
     id: "related",
     title: text_maker("related_title"),
     description: text_maker("related_desc"),
+    svg_content: { svg: IconRelatedBubble },
   },
   {
     id: "all_data",
     title: text_maker("all_data_title"),
     description: text_maker("all_data_desc"),
+    svg_content: { svg: IconDataset },
```

Comment:
Any reason to wrap these in an object? Feels like these lines could just be `svg: IconDataset,` etc.

Reply:
Earlier when I was running it without the extra object layer, it would be executed as a function immediately. I think there was some code in `infographic.js` which tried to run all functions immediately

Reply:
Ahh, haha, ok. I remember that, these are all set up to optionally be functions (I think of `subject`) rather than values, but the check of `_.isFunction` would return true for these components so they'd be miss-used. 

Well, same effect but a bit cleaner would be `svg: () => IconDataset,` I believe. A little funky, but less weird than the bubble menu having to unwrap the component from an object. Or `svg: [IconDataset]` I think, although react would likely complain about missing `key` values.

Either way, if you leave it as is or change it, please add a comment in `bubble_definitions.js` explaining the weird gotcha that these approaches are handling.

Reply:
Or perhaps better, where the bubble definitions get consumed, check `React.isValidElement()` or something alongside checking `_.isFunction`, to avoid calling the components in the wrong place. Then everything is neat and pretty self explanatory, hack-warning comments needed.